lib/src/services/controllers/like_controller.dart

```diff
@@ -0,0 +1,63 @@
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+import 'package:rescado/src/data/models/animal.dart';
+import 'package:rescado/src/data/models/like.dart';
+import 'package:rescado/src/data/repositories/card_repository.dart';
+import 'package:rescado/src/services/controllers/match_controller.dart';
+import 'package:rescado/src/utils/logger.dart';
+
+final likesControllerProvider = StateNotifierProvider<LikesController, AsyncValue<List<Like>>>(
+  (ref) => LikesController(ref.read).._initialize(),
+);
+
+class LikesController extends StateNotifier<AsyncValue<List<Like>>> {
+  static final _logger = addLogger('LikeController');
+
+  final Reader _read;
+
+  LikesController(this._read) : super(const AsyncValue.loading());
+
+  void _initialize() async {
+    _logger.d('initialize()');
+    state = const AsyncValue.loading();
+
+    fetchLikes();
+  }
+
+  Future<void> fetchLikes() async {
```

Comment:
I think this can be private.

Reply:
FetchLikes is called when refreshing likes list